transcom/mymove
# B-18668 Adding elevated rights supervisor choice

Summary
Adding a new Privileges section under the Create/Edit pages of Office Users in the Admin application. The Customer and Contracting Officer roles should not be allowed to have Supervisor privileges. This new privilege is not a required field.

Verification Steps for Reviewers

These are to be checked by a reviewer.

 Has the branch been pulled in and checked out?
 Have the BL acceptance criteria been met for this change?
 Was the CircleCI build successful?
 Has the code been reviewed from a standards and best practices point of view?
How to test

Login to the Admin application
Choose to Create a new office user
You should see a new Privileges section under the Roles checkboxes. This will show a Supervisor checkbox.
Verify that if you choose the role of Customer or Contracting Officer, you should not be able to choose the Supervisor privilege choice.
Verify that if you first check Supervisor, that you cannot choose the Customer or Contracting Officer roles checkboxes.
Verify that you can create and save a new user with the Supervisor privilege.
Choose an already existing Office user, click on the user to go into the Edit screen. Verify that all the same logic from steps 3-6 applies to this page as well.
Verify that you can export the office users and see the new privileges column in the spreadsheet.
Alter some users in the spreadsheet to now have Supervisor privileges, or remove supervisor privileges from one that have it and import the changes. Verify that you see those changes in the application now for those users.
